recipes/cuda-python/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "cuda-python" %}
+{% set version = "11.4.0" %}
+{% set major_version = version.split(".")[0] %}
+{% set minor_version = version.split(".")[1] %}
+{% set patch_version = version.split(".")[2] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1c1bd49919c8646b7d84e3e63f5a0a971b64a69a9c986194d66f17cd26ee006
+
+build:
+  number: 0
+  skip: true
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - setuptools
+    - cython
+    - pycparser
+    - autopxd2
+    - pyclibrary
+    - python
+    - pip
+    - cudatoolkit {{ major_version ~ ".*.*" }}
+  run:
+    - python
+    - {{ pin_compatible('cudatoolkit', min_pin='x', max_pin='x') }}
+  run_constrained:
+    - __glibc >=2.17  # [linux]
+
+test:
+  imports:
+    - cuda
+    - cuda.cuda
```

Comment:
Nitpick: Would this import actually work without a CUDA driver?

Reply:
It did previously ( https://github.com/conda-forge/staged-recipes/pull/16049#commits-pushed-5b0b37a ) as noted by the `linux_64_centos7` build. Unfortunately Azure has already deleted the log associated with that build